### ISSUE TITLE
comma between number of player as 2-,2 players

### ISSUE THREAD

**Issue by MaryPek** [ludobox/ludobox#94]
There is a comma between the number of players on the top of the presentation's game as  **2-,2** players

**Comment by tuxun** [comment]
damn, they come from the json :/ for now it can be filtered in the UI but it would be nice to remove them from the json files and tweak a little bit the function which create these comma when you add game

**Comment by clemsos** [comment]
No, it does not look like it is hardcoded in the data. Check an example here: https://box.ludobox.net/api/games/game-surakarta-fr The comma we see is just the regular JSON structure for object.

My wild guess is that this comes from

https://github.com/ludobox/ludobox/blob/135c9bf2d12f2671655bcf46f3a3a75e28dad06a/client/src/components/Game/GamePage.jsx#L26-L29

and expecially

https://github.com/ludobox/ludobox/blob/135c9bf2d12f2671655bcf46f3a3a75e28dad06a/client/src/components/Game/GamePage.jsx#L14-L17

which is a stylish but apparently buggy to concatenate an array to a clean list with a separator :)
